libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexAdapter.cs

```diff
@@ -58,24 +59,17 @@ public WebexAdapter(IWebexAdapterOptions config)
 
                 if (endpoint.Host != null)
                 {
-                    this.config.PublicAdress = endpoint.Host + endpoint.Port;
+                    this.config.PublicAdress = endpoint.Host;
                 }
                 else
                 {
-                    // Could not determine hostname of public address
+                    throw new Exception("Could not determine hostname of public address");
                 }
             }
             else
             {
-                // error: public_address parameter required to receive webhooks
+                throw new Exception("PublicAddress parameter required to receive webhooks");
             }
-
-            if (this.config.Secret == null)
-            {
-                // error: WARNING: No secret specified. Source of incoming webhooks will not be validated. https://developer.webex.com/webhooks-explained.html#auth
```

Comment:
we _might_ add some sort of warning for this, since without validation there's no way of knowing if messages come _actually_ from Webex